src/main/resources/openapi/audiorecording.yaml

```diff
@@ -0,0 +1,130 @@
+openapi: 3.0.1
```

Comment:
Can the spec be "merged" into the existing `src/main/resources/openapi/audio.yaml` spec? Preferably we'd have only one spec per feature.

Reply:
ACCEPT